Add GPU note on Efficiency dashboard (#1156)
kubecost/docs: using-kubecost/navigating-the-kubecost-ui/savings/gpu-optimization.md

## Known Limitations

In the first version of the GPU Optimization Savings Insights card there are a few known limitations.

- Multiple containers with the same name and running on the same cluster, node, and namespace combination (i.e., "identical" containers) might result in the following effects:
  - The savings number provided on Optimize and Remove cards may be an implicit sum of the total cost these containers.
  - Recommendations will only be provided for one of them.
  - The utilization table may not show these identical containers.
- GPU nodes must be running or have run at least one container utilizing a GPU for it to be represented on the utilization table in either the Cluster aggregation’s GPU nodes column or on the Node aggregation.

> - The Optimize recommendation may not be as accurate as possible in certain cases since Kubecost currently infers utilization about all GPUs from a single averaged utilization number.

Reviewer: ```suggestion
- The Optimize recommendation may not be as accurate as possible in certain cases since Kubecost currently infers utilization about each GPU from its averaged utilization number.
```

Would this slight change be more accurate? For me, it adds clarity to the sentence.

Reviewer 2: To me this suggests Kubecost is looking at each GPU and then determining its average utilization which isn't what's happening here. It's getting one number back which is an implicit average of all GPUs, which is what we need to improve for the next version so that it _can_ look at each GPU individually.

Reviewer: Ok I see, makes sense! Let's leave the wording as is.
